manifests/server/config.pp

```diff
@@ -29,16 +29,35 @@
     mode   => '0755',
   }
 
-  file { $cfg_file:
-    ensure  => present,
-    owner   => $owner,
-    group   => $group,
-    mode    => '0644',
-    require => [
-      File[$cfg_dir],
-      Class['dns::server::install']
-    ],
-    notify  => Class['dns::server::service'],
+  case $::osfamily {
+    'RedHat': {
+      file { $cfg_file:
+        ensure  => present,
+        owner   => $owner,
+        group   => $group,
+        mode    => '0644',
+        content => template("${module_name}/named.conf.erb"),
```

Comment:
One other error on line `dns__server__config_spec.rb`:39 - the included file needs to be in `"`-quotes:  `….with_content("/^include \"/etc/named/named.conf.options\";$/")`